wire-ios/Wire-iOS/Sources/UserInterface/SelfProfile/SelfProfileViewController.swift

```diff
@@ -162,6 +162,7 @@ final class SelfProfileViewController: UIViewController {
 
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
+        SelfProfileViewsMonitor.shared.onDidViewSelfProfile()
```

Comment:
Please don't use singletons.
Ideally we want to test the `SelfProfileViewController` individually without side effects.
For that we usually declare protocols for any dependencies.
We also use Sourcery for auto-creating the mocks.